tests/pytests/unit/modules/win_lgpo/test_get_policy.py

```diff
@@ -0,0 +1,229 @@
+import copy
+import logging
+import os
+import pathlib
+
+import pytest
+
+import salt.grains.core
+import salt.modules.win_file as win_file
+import salt.modules.win_lgpo as win_lgpo
+import salt.utils.files
+import salt.utils.win_dacl as win_dacl
+
+log = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.slow_test,
+]
+
+
+@pytest.fixture
+def configure_loader_modules(tmp_path):
+    cachedir = tmp_path / "__test_admx_policy_cache_dir"
+    cachedir.mkdir(parents=True, exist_ok=True)
+    return {
+        win_lgpo: {
+            "__salt__": {
+                "file.file_exists": win_file.file_exists,
+                "file.makedirs": win_file.makedirs_,
+            },
+            "__opts__": {
+                "cachedir": str(cachedir),
+            },
+        },
+        win_file: {
+            "__utils__": {
+                "dacl.set_perms": win_dacl.set_perms,
+            },
+        },
+    }
+
+
+@pytest.fixture(scope="module")
+def osrelease():
+    grains = salt.grains.core.os_data()
+    yield grains.get("osrelease", None)
+
+
+@pytest.fixture
+def clean_comp():
+    reg_pol = pathlib.Path(
+        os.getenv("SystemRoot"), "System32", "GroupPolicy", "Machine", "Registry.pol"
+    )
+    reg_pol.unlink(missing_ok=True)
+    yield reg_pol
+    reg_pol.unlink(missing_ok=True)
+
+
+@pytest.fixture
+def checkbox_policy():
+    policy_name = "Configure Corporate Windows Error Reporting"
+    policy_settings = {
+        "Connect using SSL": False,
+        "Corporate server name": "fakeserver.com",
+        "Only upload on free networks": False,
+        "Server port": 1273,
+    }
+    win_lgpo.set_computer_policy(name=policy_name, setting=copy.copy(policy_settings))
+    yield policy_name, policy_settings
+    win_lgpo.set_computer_policy(name=policy_name, setting="Not Configured")
```

Comment:
```suggestion
    win_lgpo.set_computer_policy(name=policy_name, setting=copy.copy(policy_settings))
    try:
        yield policy_name, policy_settings
    finally:
        win_lgpo.set_computer_policy(name=policy_name, setting="Not Configured")
```
That last function should run even on errors, right?